test/resources/examples/index.js

```diff
@@ -262,6 +262,51 @@ const shortRingBetterTrade = generateTestCase({
   ],
 })
 
+const maxUint128 = new BN(2).pow(new BN(128)).sub(new BN(1))
+const exampleNoUsedAmountTracking = generateTestCase(
+  {
+    deposits: [
+      { amount: feeAdded(toETH(5)), token: 0, user: 0 },
+      { amount: feeAdded(toETH(5)), token: 1, user: 1 },
+    ],
+    orders: [
+      {
+        sellToken: 0,
+        buyToken: 1,
+        sellAmount: maxUint128,
+        buyAmount: feeSubtracted(maxUint128.div(new BN(4))),
+        user: 0,
+      },
+      {
+        sellToken: 1,
+        buyToken: 0,
+        sellAmount: toETH(10),
+        buyAmount: feeSubtracted(toETH(10), 2),
+        user: 1,
+      },
+      {
+        sellToken: 0,
+        buyToken: 1,
+        sellAmount: maxUint128,
+        buyAmount: feeSubtracted(maxUint128.div(new BN(4))),
+        user: 0,
+      },
+    ],
+    solutions: [
+      {
+        prices: [1, 1].map(toETH),
+        buyVolumes: [toETH(3), feeSubtracted(toETH(3)), ZERO],
+      },
+      {
+        prices: [1, 1].map(toETH),
+        buyVolumes: [ZERO, feeSubtracted(toETH(5)), toETH(5)],
+      },
+    ],
+  },
+  false,
+  true
```

Comment:
Can we remove these! They are only for debugging purposes... Also, requiring `strict = false` might result in a token conservation issue.

Reply:
Actually, `strict = false` is needed. 
It is needed as the disregarded utility can not be calculated by the js scripts, if the `sellVolume > sellBalance`. There is this `todo` in the code.

Reply:
Ok, then maybe this is why my example were failing here.